storage/go110_test.go

```diff
@@ -19,6 +19,7 @@ package storage
 import (
 	"context"
 	"errors"
+	"golang.org/x/xerrors"
```

Comment:
The linting is failing because this import should come after the standard library ones. It should go before L27. You can run `goimports` to correct.